README.md

```diff
@@ -4,32 +4,65 @@
 
 ## Dependencies
 ### Supported Databases
-Holmes-Storage supports multiple databases and splits them into two categories: Object Stores and Document Stores. This was done to provide users to more easily select their preferred solutions while also allowing the mixing of databases for optimization purposes. In production environments, we strongly recommend using an [S3](https://aws.amazon.com/documentation/s3/) compatible Object Store, such as [RIAK-CS](http://docs.basho.com/riak/cs/latest), and a clustered deployment of [Cassandra](http://cassandra.apache.org/) for the Document Store. 
+Holmes-Storage supports multiple databases and splits them into two categories: Object Stores and Document Stores. 
+Object Stores are responsible for storing the file-based malicious objects collected by the analyst: binary files such as PE32 and ELF, PDFs, HTML code, Zips files etc. Document Stores are contain the results of the Holmes-Totem and Holmes-Totem-Dynamic analytic results of the files in Object Stores. This was done to enable users to more easily select their preferred solutions while also allowing the mixing of databases for optimization purposes. In production environments, we strongly recommend using an [S3](https://aws.amazon.com/documentation/s3/) compatible Object Store, such as [RIAK-CS](http://docs.basho.com/riak/cs/latest), and a clustered deployment of [Cassandra](http://cassandra.apache.org/) for the Document Stores.
 
-#### Object Stores
-We support three primary object storage databases. We recommend S3 for large deployments.
+#### I. Object Stores
+We support three primary object storage databases. 
 - Local File System
 - S3 compatible
 - (Soon) MongoDB Gridfs
 
-#### Document Stores
-We support two primary object storage databases. We recommend a Cassandra cluster for large deployments.
+There are several tools you can use for implementing Object Stores. Depending on the intended scale of your work with Holmes-Storage, we would recommend:
+
+- [AWS](https://aws.amazon.com/) for cloud computing, industry and large-scale implementation
```

Comment:
might be good to have this as a table. Maybe we could have columns  for workstation, midsized, and large-scale